hack/deploy-karmada-by-operator.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Karmada Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+
+# This script deploy karmada control plane to any cluster you want via karmada-operator. REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# This script depends on utils in: ${REPO_ROOT}/hack/util.sh
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
+CERT_DIR=${CERT_DIR:-"${HOME}/.karmada"}
+KARMADA_VERSION=$(curl -sl https://api.github.com/repos/karmada-io/karmada/releases |grep tag_name | awk '{print $2}'| sed -n '1p'| awk -F '-' '{print $1}'| tr -d 'v"')
+mkdir -p "${CERT_DIR}" &>/dev/null ||  mkdir -p "${CERT_DIR}"
+rm -f "${CERT_DIR}/*" &>/dev/null ||  rm -f "${CERT_DIR}/*"
+
+source ${REPO_ROOT}/hack/util.sh
+
+function usage() {
+  echo "This script deploys karmada control plane components to a given cluster via karmada-operator."
+  echo "Note: This script is an internal script and is not intended used by end-users."
+  echo "Usage: hack/deploy-karmada-by-operator.sh <KUBECONFIG> <CONTEXT_NAME> <ADDON_NEEDED> <CRD_LOCAL_PATH>"
+  echo "Example: hack/deploy-karmada.sh ~/.kube/config karmada-host true"
```

Comment:
The example seems to be wrong.

Reply:
`CRD_LOCAL_PATH` is an optional parameter

Reply:
```suggestion
  echo "Example: hack/deploy-karmada-by-operator.sh ~/.kube/config karmada-host true"
```

here's what i mean.